pkgs/applications/networking/utahfs/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, lib, fetchFromGitHub }:
```

```suggestion
{ lib, buildGoPackage, fetchFromGitHub }:
```
`stdenv` is not used